src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -605,7 +625,7 @@ private static NullableAnnotation MergeNullableAnnotation(NullableAnnotation a,
 
         public string ToDisplayString(SymbolDisplayFormat format = null)
         {
-            var str = TypeSymbol.ToDisplayString(format);
+            var str = (TypeSymbol is null) ? "<null>" : TypeSymbol.ToDisplayString(format);
```

Comment:
>"<null>" [](start = 45, length = 8)

Depending on earlier question (what annotations are allowed when `TypeSymbol` is null), we should consider printing the annotation here.

Reply:
We do.  This is just the string for the type part.  We add the annotation in the next if statement.

---
In reply to: [260045795](https://github.com/dotnet/roslyn/pull/33648#discussion_r260045795) [](ancestors = 260045795)